ui/src/app/core/job-manager.service.ts

```diff
@@ -9,6 +9,7 @@ import {QueryJobsResponse} from '../shared/model/QueryJobsResponse';
 import {JobMetadataResponse} from '../shared/model/JobMetadataResponse';
 import {UpdateJobLabelsRequest} from "../shared/model/UpdateJobLabelsRequest";
 import {UpdateJobLabelsResponse} from "../shared/model/UpdateJobLabelsResponse";
+import { TimeFrame, AggregationResponse } from '../shared/model/models';
```

Comment:
nit: match spacing from the other imports

Reply:
fixed.